src/openfermion/ops/__init__.py

```diff
@@ -11,6 +11,7 @@
 #   limitations under the License.
 
 from ._binary_polynomial import BinaryPolynomial
+from ._diagonal_coulomb_hamiltonian import DiagonalCoulombHamiltonian
```

Comment:
I am not sure I like "DiagonalCoulombHamiltonian" because it seems to exclude other operators that might not have any explicit connection to Coulomb operators or Hamiltonians, which also fit this form. Sorry if I agreed to this name in the past...

what about DiagonalTwoBodyOperator?

Reply:
Maybe this name is okay. If you like, let's just keep it.

Reply:
I had the same thought, but I think DiagonalTwoBodyOperator sounds too much like it doesn't contain one-body terms.